blocklist-client/src/logging.rs

```diff
@@ -11,7 +11,6 @@ use tracing_subscriber::EnvFilter;
 /// # Arguments
 ///
 /// * `pretty` - A boolean that determines if the logging format should be pretty or JSON
```

Comment:
Locally I needed to change the * to a - otherwise cargo fmt re-added a newline here

Reply:
🙏🏽 , will update

Reply:
Hmmm, didn't happen for me, but updated anyway in https://github.com/stacks-network/sbtc/pull/1213/commits/2f0b55ca5d20f9788d083ed3bc3e1d507cb38081.